src/chains/mainnet/vm/opcodes/block/number.ts

```diff
@@ -0,0 +1,27 @@
+import { Opcode } from '@/chains/types';
+
+export const number: Opcode = {
+  number: 43,
```

Comment:
All opcode numbers should be written as `number: 0x43` instead of `number: 43` (JS supports writing hex numbers that way). Since NUMBER is really 0x43 = 67

Reply:
Oh good catch, this was confusing